### ISSUE TITLE
Test failures with perl < 5.26

### ISSUE THREAD

**Issue by eserte** [team-at-cpan/Acme-Signature-Arity#1]
See subject.
For perl < 5.20:
```
Need perl 5.20.0 or later for feature signatures
```
For perl 5.20 .. 5.24:
```
    #   Failed test 'max arity is 0'
    #   at t/arity.t line 23.
    #          got: undef
    #     expected: '0'
Undefined subroutine &main::arity called at t/arity.t line 23.
    # Child (sub () { }) exited without calling finalize()

#   Failed test 'sub () { }'
#   at /usr/share/perl/5.24/Test/Builder.pm line 279.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 29 just after 1.
t/arity.t .............. 
Dubious, test returned 29 (wstat 7424, 0x1d00)
Failed 1/1 subtests 
```

**Comment by tm604** [comment]
Thanks - have updated minimum Perl version to 5.26+, since this module is unlikely to work with earlier signature implementations anyway.

**Comment by tm604** [comment]
Should be fixed in adc6c6c2d7923748d4d871c35fce9824c33d9b3a released as 0.003.
